CTP/sql/src/com/navercorp/cubridqa/cqt/console/dao/ConsoleDAO.java

```diff
@@ -298,14 +298,28 @@ private Connection createConnection(String db) {
 	}
 
 	public void execute(Connection conn, Sql sql, boolean isPrintQueryPlan) {
+		this.onMessage("sql.script: " + sql.getScript());
 		this.onMessage("sql.getType: " + sql.getType());
+
 		if (sql.getType() == Sql.TYPE_CALL) {
 			executeCall(conn, sql);
 		} else if (sql.getType() == Sql.TYPE_PRE_STMT) {
 			executePrepareStatement(conn, sql, isPrintQueryPlan);
 		} else {
 			executeStatement(conn, sql, isPrintQueryPlan);
 		}
+
+		// Print Output messages
+		if (test.getServerOutput().equalsIgnoreCase("on")) {
+			ArrayList<SqlParam> params = new ArrayList<SqlParam>();
+			params.add(new SqlParam("OUT", 1, null, Types.VARCHAR));
+			params.add(new SqlParam("OUT", 2, null, Types.INTEGER));
+			Sql getLine = new Sql(test.getConnId(), "CALL GET_LINE (?, ?);", params, true);
```

Comment:
This getLine can be created in the getServerOutputMessage() method rather than be handed over as an argument.

Reply:
Fixed at 78bbe27